tutorials/integrations/integrations/pytorch/timm/model_lib_timm.py

```diff
@@ -0,0 +1,54 @@
+import timm
+from timm.data import create_dataset, create_loader, resolve_data_config
+
+from integrations.common.base_classes import BaseModelLib
+from integrations.pytorch.helpers import classification_eval, get_representative_dataset
+from integrations.common.consts import MODEL_NAME, BATCH_SIZE, VALIDATION_SET_LIMIT, VALIDATION_DATASET_FOLDER
+
+import logging
+
+
+
+class ModelLib(BaseModelLib):
+
+    def __init__(self, args):
+        model_list = timm.list_models('')
+        if args[MODEL_NAME] not in model_list:
+            logging.error(f'Unknown model, Available timm models : {model_list}')
+        self.model = timm.create_model(args[MODEL_NAME], pretrained=True)
```

Comment:
What about the model name is a model from timm that is not classification ? this code can crash.

Reply:
You are right. But I think timm is only classification... I'm trying to re-confirm